### ISSUE TITLE
Warning

### ISSUE THREAD

**Issue by beckynevin** [deepskies/DeepUQ#193]
Use .clone().detach() instead of torch.tensor() [applies to data.py]

**Comment by beckynevin** [comment]
--> a better fix is torch.tensor() instead of torch.Tensor() and then specify dtype

**Comment by beckynevin** [comment]
Addressed in PR #197
